services/order.py

```diff
@@ -0,0 +1,37 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                movie_session_id=ticket["movie_session"],
+                order_id=order.id,
+                row=ticket["row"],
+                seat=ticket["seat"]
+            )
+
+        return order
+
+
+def get_orders(username: str = None) -> QuerySet[Order]:
+    orders = Order.objects.all().prefetch_related("user")
```

Comment:
read checklist in readme, you don't need all(). here

Reply:
Thank you. Was constantly worried about electricity, so made a couple of stupid mistakes :)